.github/workflows/ci-precompiled.yaml

```diff
@@ -0,0 +1,111 @@
+# Copyright 2024 NVIDIA CORPORATION
```

Comment:
I created a new .github/workflows/ci-precompiled.yaml file that can be ported or appended to the pre-compiled scheduled ci file